source/readout_panel_example.hpp

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include <string>
+
+#include "emp/math/Random.hpp"
+#include "emp/prefab/Card.hpp"
+#include "emp/prefab/CodeBlock.hpp"
+#include "emp/prefab/ReadoutPanel.hpp"
+#include "emp/web/Button.hpp"
+#include "emp/web/Document.hpp"
+
+int counter = 0;
+
+void readout_panel_example( emp::web::Document& doc ) {
+
+  // ------ Readout Panel Example ------
+  emp::prefab::Card readout_panel_ex("INIT_CLOSED");
+  doc << readout_panel_ex;
+  readout_panel_ex.AddHeaderContent("<h3>Readout Panel</h1");
+  readout_panel_ex << "<h3>Live Demo:</h3><hr>";
+
+  emp::prefab::ReadoutPanel values("Readout Values", 100); // Refreshes 10 times a second
+
+  // A random number generator
+  std::function<std::string()> random_number = [=]() mutable {
```

Comment:
Does this need mutable?? Is it capturing anything ??

Reply:
It might need to me mutable because of the Random instance inside but no it's not capturing everything so I'll fix that.

Reply:
You were also right about mutable, it doesn't need to be.